packages/boilerplate-generator/boilerplate_web.browser.html

```diff
@@ -21,8 +21,10 @@
 {{/each}}
 
 {{{head}}}
+{{{dynamicHead}}}
```

Comment:
I think it'd be nice to have `dynamicHeadBefore` and `dynamicHeadAfter` for injecting stuff at the beginning of the head or at the end. `dynamicHead` only injects at the end. And same with the body.